catalog/mvcc/cataloger_delete_entry_test.go

```diff
@@ -150,8 +151,11 @@ func TestCataloger_DeleteEntryAndCheckItRemainsInCommits(t *testing.T) {
 	if err != nil {
 		t.Fatal("Failed to commit after delete:", err)
 	}
-	entry, err = c.GetEntry(ctx, repository, nextCommit.Reference, "/file2", catalog.GetEntryParams{})
-	entry, err = c.GetEntry(ctx, repository, prevCommit.Reference, "/file2", catalog.GetEntryParams{})
+	_, err = c.GetEntry(ctx, repository, nextCommit.Reference, "/file2", catalog.GetEntryParams{})
+	testutil.Must(t, err)
```

Comment:
Shouldn't the file be missing here?  Indeed, the test appears to fail.

Reply:
Will verify that the file is missing